Lib/fontbakery/profiles/googlefonts_conditions.py

```diff
@@ -686,7 +686,7 @@ def production_metadata():
     """Get the Google Fonts production metadata"""
     import requests
     meta_url = "http://fonts.google.com/metadata/fonts"
-    return requests.get(meta_url).json()
+    return requests.get(meta_url, timeout=10).json()  # XXX
```

Comment:
`config.get("timeout")`

Reply:
But that doesn’t work because we don’t have config here.